smithy-model/src/main/resources/software/amazon/smithy/model/loader/prelude-traits.smithy

```diff
@@ -0,0 +1,516 @@
+$version: "0.3.0"
+
+namespace smithy.api
+
+/// Makes a shape a trait.
+@trait(selector: ":each(simpleType, list, map, set, structure, union)")
```

Comment:
Should this conflict with `error`?

Reply:
Error marks it as conflicting, which is transitive.